email/1.3/modules/ROOT/pages/email-gmail.adoc

```diff
@@ -89,17 +147,19 @@ The way you troubleshoot SMTPS connection issues depends on whether or not you h
 
 If you have two-factor authentication set up:
 
-. There is no need for you to enable *Less Secure Apps*.
-. Generate an app-specific password and use that instead of your normal password. +
+* There is no need for you to enable *Less Secure Apps*.
```

Comment:
This is a style comment, but the text in bullets is not consistent in terms of structure. I suggest:

If you have two-factor authentication set up, generate an app-specific password and use that instead of your normal password. There is no need to enable *Less Secure Apps*.